packages/webpack-configs/src/defineConfig.ts

```diff
@@ -168,7 +168,7 @@ export function defineDevHostConfig(swcConfig: SwcConfig, applicationName: strin
     const {
         entry = path.resolve("./src/index.ts"),
         publicPath = "auto",
-        cache = false,
+        cache = true,
```

Comment:
It's true by default so you don't need to specifically specify a value.

Reply:
Fixed